MSAL/test/integration/native_auth/end_to_end/reset_password/MSALNativeAuthResetPasswordEndToEndTests.swift

```diff
@@ -25,24 +25,28 @@
 import Foundation
 import XCTest
 
-final class MSALNativeAuthResetPasswordEndToEndTests: MSALNativeAuthEndToEndBaseTestCase {
-
-    private let usernameOTP = ProcessInfo.processInfo.environment["existingOTPUserEmail"] ?? "<existingOTPUserEmail not set>"
-    
-    // Hero Scenario 2.3.1. SSPR – without automatic sign in
+final class MSALNativeAuthResetPasswordEndToEndTests: MSALNativeAuthEndToEndPasswordTestCase {
```

Comment:
I think this class can extend `MSALNativeAuthEndToEndBaseTestCase` instead of `MSALNativeAuthEndToEndPasswordTestCase`

Reply:
Yes, done.

Reply:
Done.